src/librustc/infer/mod.rs

```diff
@@ -264,8 +264,8 @@ pub enum SubregionOrigin<'tcx> {
     /// Dereference of reference must be within its lifetime
     DerefPointer(Span),
 
-    /// Closure bound must not outlive captured free variables
-    FreeVariable(Span, ast::NodeId),
+    /// Closure bound must not outlive captured variables
+    ClosureCapture(Span, ast::NodeId),
```

Comment:
Slight inconsistency that this is changed to `Capture` directly, but if we're moving there anyway (which I support), then ok by me.

Reply:
I think I did this because the comment and the user-facing message this causes both contain the word "capture".

Reply:
`UpvarCapture` would've also probably worked, but this seemed nicer.